gui/__tests__/containers/SignIn.test.js

```diff
@@ -59,8 +59,8 @@ describe('containers/SignIn', () => {
     await syncFetch(wrapper)
 
     const newState = store.getState()
-    expect(newState.errors).toEqual({
-      messages: ['Your email or password is incorrect. Please try again'],
+    expect(newState.routing).toEqual({
```

Comment:
In this initial implementation I felt the need to have errors in each state group because global errors kept rerendering the whole Layout and clobbering the input textarea.

What initially happened
1. `Layout.js` listens to global errors.
2.  Submitting `Job/BridgeForm` adds this new unauthorized, sign back in error.
3. Triggers a rerender of layout where it realizes its authentication.allowed is false
4. Renders sign in page.

What happens now
1. `Layout.js` only listens to `routing.errors` (we can rename this) and works as expected
2. `Job/BridgeForm` only listens to `create.errors`
3.  Submitting `Job/BridgeForm` adds this new unauthorized, sign back in error.
4. `Job/BridgeForm` only rerenders `FormNotifications` and doesn't clobber input textarea. 

Potential solution
1. `Layout.js` listens to global errors.
2.  Submitting `Job/BridgeForm` adds this new unauthorized, sign back in error.
3. We keep authentication.allowed true to prevent redirect to sign i page.
4. Rerender of Layout keeps content.

I'm open to other ideas. I'm going to think on this a bit myself.